gamemode/modules/items/sh_items.lua

```diff
@@ -92,6 +92,52 @@ items.items = {
         viewModelOffset = Vector( -2, 5, -9 ),
         viewModelAng = Angle( 0, -15, 0 )
     },
+    {
+        type = "storage_box",
+        displayName = "Storage Box",
+        description = "Store all your items!",
+        maxCount = 1,
+        model = "models/props_junk/wood_crate001a.mdl",
+        icon = "materials/rvr/items/water_bottle.png", -- TODO proper icon?
+        placeable = true,
+        placeableClass = "rvr_storage",
+        viewModelOffset = Vector( 30, 50, -9 )
+    },
+    {
+        type = "workbench",
+        displayName = "Work Bench",
+        description = "Crafting.. woah!",
+        maxCount = 1,
+        model = "models/rvr/props/workbench.mdl",
+        icon = "materials/rvr/items/water_bottle.png", -- TODO proper icon?
+        placeable = true,
+        placeableClass = "rvr_crafter",
+        viewModelOffset = Vector( 30, 60, -39 ),
+        viewModelAng = Angle( 0, -40, 0 )
+    },
+    {
+        type = "water_purifier",
+        displayName = "Water Purifier",
+        description = "Makes water from air? is this magick?",
+        maxCount = 1,
+        model = "models/props_borealis/bluebarrel001.mdl", -- TODO a better model?
+        icon = "materials/rvr/items/water_bottle.png", -- TODO proper icon
+        placeable = true,
+        placeableClass = "rvr_water_purifier",
+        viewModelOffset = Vector( 30, 50, -9 )
+    },
+    {
+        type = "grill",
+        displayName = "Grill",
+        description = "Cook your food. Or eat it raw...",
```

Comment:
```suggestion
        description = "Cook your food. Or eat it, rawr~",
```
at @samuelWilliams99 request